app/views/movies/_form.html.erb

```diff
@@ -11,9 +11,29 @@
   <% end %>
 
   <%= form.label :title %>
-  <%= form.text_field :title %></br></br>
+  <%= form.text_field :title %>
+  <div class = 'clear-fix'></div>
```

Comment:
I think the Bootstrap's class name is `clearfix`

Reply:
yes sir you are right, i'll change that